rms/file_processing/services.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import UploadFile, HTTPException
+from rms.file_processing.models import FileUploadResult
+from rms.file_processing.clients import AzureBlobClient
+from rms.file_processing.managers import FileManager
+from datetime import datetime
+
+async def upload_to_storage(file: UploadFile, file_name: str) -> FileUploadResult:
+    blob_client = AzureBlobClient()
+
+    try:
+        file_path = await blob_client.upload(file, file_name)
+    except Exception as e:
+        print(e)
+        raise HTTPException(status_code=500, detail=f"An error occurred while uploading the file: {str(e)}")
```

Comment:
A more semantically correct error code is 400, because 5xx should be reserved for unseen system failures on our side

Reply:
fixed